src/index.html

```diff
@@ -6,9 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="nav-container">
+      <a href="#home" class="homeLink">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav-list">
+        <a href="#apple" class="is-active nav-link" >apple</a>
+        <a href="#samsung" class="nav-link">samsung</a>
+        <a href="#smartphones" class="nav-link">smartphones</a>
```

Comment:
For consistency and readability, consider capitalizing the first letter of the link text. For example, change 'smartphones' to 'Smartphones'.